A tutorial on how to add a planner plugin to MoveIt (template) (#331)
ros-planning/moveit_tutorials: doc/controller_configuration/controller_configuration_tutorial.rst

Example Controller Manager
--------------------------

> An example is provided in plugin tutorial at `Creating MoveIt Plugins <../creating_moveit_plugins/plugin_tutorial.html>`_.

Reviewer: ```suggestion
See the example provided in the `Creating MoveIt Plugins <../creating_moveit_plugins/plugin_tutorial.html>`_ tutorial.
```
